src/dlgTriggerEditor.h

```diff
@@ -432,10 +433,25 @@ private slots:
     void setupPatternControls(const int type, dlgTriggerPatternEdit* pItem);
     void key_grab_callback(const Qt::Key, const Qt::KeyboardModifiers);
     void setShortcuts(const bool active = true);
+    void setShortcuts(QList<QAction*> actionList, const bool active = true);
 
     void showOrHideRestoreEditorActionsToolbarAction();
     void showOrHideRestoreEditorItemsToolbarAction();
 
+    std::unordered_map<QString, QString> expectedTextsAndShortcuts = {
```

Comment:
This needs to start with a lowercase `m` since it's a class member, suggest something along the lines of `mButtonShortcuts`

Reply:
Why are we introducing *another* container type? Why not use the familiar `QMap<T1, T2>` or `QHash<T1, T2>` where you do not have to do a `find` to check for a value separate from the assignment? It can't be that significant in performance terms given how few elements there are in the container...

Also, at what point in the execution are the `QObject::tr(...)` calls in the declarations done here? Is it when the each instance of the class is constructed and have the translations been loaded at that point?

Reply:
I'm fine with standard containers because at this point the standard is evolving faster than Qt is with all of the improvements that are happening.

`tr()` - right

Reply:
I did not choose this container type for performance reasons as you assume, and I am not very familiar with any of the three container types at all, so I have few preferences. This evolved from a simple code (repetition, then struct) to loop over into a map type for quicker lookup, so should actually improve performance.